sample/android-app-compose/build.gradle.kts

```diff
@@ -6,19 +6,17 @@ plugins {
     id("android-app-convention")
     id("dev.icerock.mobile.multiplatform-resources")
     id("kotlin-android")
+    id("org.jetbrains.compose")
```

Comment:
in android we need jetbrains compose? it realy needed?

Reply:
As I see in their samples it is the recommended way.